WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -364,31 +364,6 @@ private extension StorePickerViewController {
         ServiceLocator.authenticationManager.presentSupport(from: self, screen: .storePicker)
     }
 
-    func presentAddStoreActionSheet(from view: UIView) {
-        let actionSheet = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
-        actionSheet.view.tintColor = .text
-        let createStoreAction = UIAlertAction(title: Localization.createStore, style: .default) { [weak self] _ in
-            // TODO: add tracks for site creation
-            self?.createStoreButtonPressed()
```

Comment:
The method `createStoreButtonPressed` should be removed as well. And the method has a tracking event `.sitePickerCreateSiteTapped` which should probably removed.

And how about the `StorePickerViewControllerDelegate` protocol's `func createStore()` method which will no longer be used? 🤔

Reply:
I deleted the method/tracking/createStore() function in https://github.com/woocommerce/woocommerce-ios/pull/12376/commits/8a4c1db13b814093e296196643edae5507f7cf40

As I looked into this, there are a lot more things that's technically can be deleted (e.g: more trackings). But, it also feels like it will make the scope wider than the PR expected.

My thinking is we can just delete the entry point for now, and remove all the unused code later on. This seems like the same thing done in Android, where they have a specific PR for code cleanup that deletes the majority of things:

Reply:
Good question. 

I would prefer removing the code now if it is straightforward and quick. The rest of the code we can leave it for the "clean up" PR. Please feel free to follow any approach that suits you. 

As a side note, I think it would be better to note down the code to be removed (as subtasks of the clean-up task, if possible).

Reply:
Thanks, this makes sense.

Looking at it again, the rest of the analytics are tied to store creation from prologue, so I'll leave them for now since removing them requires also removing different parts of the code.